android/src/main/kotlin/com/amplitude/amplitude_flutter/AmplitudeFlutterPlugin.kt

```diff
@@ -66,7 +65,7 @@ class AmplitudeFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
                 amplitude = Amplitude(configuration)
 
                 // Set library
-                amplitude.add(FlutterLibraryPlugin())
+                amplitude.add(FlutterLibraryPlugin(call.argument<String>("library")!!))
```

Comment:
nit: `!!` could maybe throw an error? what about this instead?
```suggestion
                amplitude.add(FlutterLibraryPlugin(call.argument<String>("library") ?? "amplitude-flutter/unknown"))
```

Reply:
I like it! Update both Kotlin and Swift at https://github.com/amplitude/Amplitude-Flutter/pull/176/commits/4eb9b6fa273ba26d7fe7d69ce1cb29aac5e33037